board/freescale/mx6ullevk/mx6ullevk_siklu_api.c

```diff
@@ -6,7 +6,7 @@
  */
 
 
-
+#ifndef CONFIG_SPL_BUILD
```

Comment:
I think that files that do not build for SPL should be excluded at the `Makefile` level.

Reply:
Yes, I thought about this. There are some platforms that do this. I'll write this as a Jira issue.